pom.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this needed in the repo root? It looks like you copied the files under `e2e/` but didn't remove the originals?